packages/product-feed-heureka/.travis.yml

```diff
@@ -16,7 +16,5 @@ install:
 
 script:
   - php vendor/bin/parallel-lint ./src ./tests
-  - php vendor/bin/php-cs-fixer fix --config=vendor/shopsys/coding-standards/build/phpcs-fixer.php_cs --dry-run --verbose --diff ./src ./tests
-  - php vendor/bin/phpcs --standard=vendor/shopsys/coding-standards/rulesetCS.xml --extensions=php --encoding=utf-8 --tab-width=4 -sp ./src ./tests
-  - php vendor/bin/phpmd "./src,./tests" text vendor/shopsys/coding-standards/rulesetMD.xml --extensions=php
+  - php vendor/bin/ecs check --config=vendor/shopsys/coding-standards/easy-coding-standard.yml --verbose ./src ./tests
```

Comment:
I think this will fail because the config doesn't contain skips for this package.

I have a suggestion - change the command to
`php vendor/bin/ecs check --config=autoload-coding-standard.yml --verbose ./src ./tests`
And add import to `autoload-coding-standard`

Or maybe it is possbile to set multiple configs.

But it is maybe not possible because monorepo will include these `.yml` files... Not an easy task, maybe contact a college.
In this case it would be possible to have two configs in the package. One with exceptions (that is already here) and one with imports of `coding-standards` packages' config and exceptions config. But I would prefer to have only one config here, because two configs are strange.

Reply:
And similar situation is in other packages

Reply:
> And add import to autoload-coding-standard

:+1: